src/plugins/visualizations/common/expression_functions/range.ts

```diff
@@ -7,7 +7,7 @@
  */
 
 import { i18n } from '@kbn/i18n';
-import { ExpressionFunctionDefinition, Datatable, Range } from '../../../expressions/public';
+import { ExpressionFunctionDefinition, Datatable, Range } from '../../../expressions/common';
```

Comment:
I think that we should also update the kibana.json file 
```
"extraPublicDirs": ["common/constants"],
```
It only declares as public the constants file and not the expression functions.
I can see this error on the server

```
ERROR in ./public/pie_fn.ts
   │ proc [kibana]        │          Module not found: Error: import [../../visualizations/common] references a non-public export of the [visualizations] bundle and must point to one of the public directories: [public, common/constants]
   │ proc [kibana]        │           @ ./public/pie_fn.ts 9:0-62 119:23-38
   │ proc [kibana]        │           @ ./public/plugin.ts
   │ proc [kibana]        │           @ ./public/index.ts
   │ proc [kibana]        │           @ /Users/stratoulakalafateli/Documents/Forks/kibana/node_modules/@kbn/optimizer/target/worker/entry_point_creator.js
```